features/models.py

```diff
@@ -74,3 +74,42 @@ def __str__(self):
     @property
     def ahti_id(self):
         return f"{self.source_type.system}:{self.source_type.type}:{self.source_id}"
+
+
+class Image(models.Model):
+    feature = models.ForeignKey(
+        Feature,
+        on_delete=models.CASCADE,
+        related_name="images",
+        verbose_name=_("feature"),
+    )
+    url = models.URLField(verbose_name=_("url"), max_length=2000)
+    copyright_owner = models.CharField(
+        verbose_name=_("copyright owner"), max_length=200
+    )
+    license = models.ForeignKey(
+        "License",
+        on_delete=models.CASCADE,
+        related_name="images",
+        verbose_name=_("license"),
+    )
+
+    class Meta:
+        verbose_name = _("image")
+        verbose_name_plural = _("images")
+        ordering = ("id",)
+
+    def __str__(self):
+        return self.url
+
+
+class License(models.Model):
+    name = models.CharField(verbose_name=_("name"), max_length=200)
```

Comment:
just the name without the license text?

Reply:
It's currently not 100% how the license text part will be handled. Current plan would be to have a url to the license, but I'll add the url field once it's clear where we can get the url. I guess it's also possible that we can (maybe in addition to the url) have a field for the license text. I made the field translatable as per the initial DB design.

Reply:
ok, well, it is a good solution for the MVP 👍